canonical/ubuntu-pro-client
# timer: only run when attached (SC-1304)

no-lp no-gh
The final PR of my 23.04 tech-debt ticket.

After our recent improvements, we only need an intermittent job to run when attached. So let's not waste everyone's cpu cycles by leaving the timer on when unattached.

## Test Steps

Verify the timer is off when upgrading to this version while unattached.
Verify the timer stays on when upgrading to this version while attached.
Verify that attaching starts the timer.
Verify that detaching stops the timer.

## Checklist


## Does this PR require extra reviews?
